setup.cfg

```diff
@@ -0,0 +1,41 @@
+[metadata]
+name = elitech
+version = 0.0.1
+description = Library and command line tool for Elitech data logger
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Pascal COMBES
+author_email = pascom@orange.fr
+license = GPL-3.0
+license_file = LICENSE
+classifiers =
+	License :: OSI Approved :: GPL License
+	'Programming Language :: Python',
+	'Programming Language :: Python :: 2',
+	'Programming Language :: Python :: 2.6',
+	'Programming Language :: Python :: 2.7',
+	'Programming Language :: Python :: 3',
+	'Programming Language :: Python :: 3.2',
+	'Programming Language :: Python :: 3.3',
+	'Programming Language :: Python :: 3.4',
+	'Programming Language :: Python :: 3.5',
+	'Programming Language :: Python :: 3.6',
+	'Programming Language :: Python :: 3.7',
+	'Programming Language :: Python :: 3.8',
+	'Programming Language :: Python :: Implementation',
+	'Programming Language :: Python :: Implementation :: CPython',
+	'Programming Language :: Python :: Implementation :: IronPython',
+	'Programming Language :: Python :: Implementation :: Jython',
+	'Programming Language :: Python :: Implementation :: PyPy',
+url = https://github.com/pasccom/python-elitech
+
+[options]
+packages = find:
```

Comment:
As is, this includes the tests and PythonUtils in the package. Also HIDParser is not included even though it is needed (but maybe it is because it is another package with `setup.cfg` and `setup.py` files).  
NB: HIDParser is needed, because it is a patched version, I did not do personnally a PR, but there is one for the same patch already.

Reply:
HIDParse will now be included in the package automatically, if it exists in the sub folder.